presto-main/src/main/java/com/facebook/presto/sql/gen/CompilerOperations.java

```diff
@@ -67,7 +67,8 @@ public static boolean in(Object value, Set<?> set)
     public static boolean testMask(@Nullable Block masks, int index)
     {
         if (masks != null) {
-            return BOOLEAN.getBoolean(masks, index);
+            boolean isNull = masks.mayHaveNull() && masks.isNull(index);
```

Comment:
Maybe just inline this line with the next since the isNull is only used once?

Reply:
I was trying to inline it, but it was less readable because of the negation.